docs/partial_source/deep_dive/17_ivy_frontends_tests.rst

```diff
@@ -200,41 +197,45 @@ ivy.tan()
 
 .. code-block:: python
 
-    # ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py
-    #tan
+    # ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py
+    # threshold
     @handle_cmd_line_args
     @given(
-        dtype_and_x=helpers.dtype_and_values(
+        dtype_and_input=helpers.dtype_and_values(
             available_dtypes=helpers.get_dtypes("float"),
         ),
         num_positional_args=helpers.num_positional_args(
-            fn_name="functional.frontends.torch.tan"
+            fn_name="ivy.functional.frontends.torch.threshold"
         ),
     )
-    def test_torch_tan(
-        dtype_and_x,
+    def test_torch_threshold(
+        dtype_and_input,
         as_variable,
-        with_out,
         num_positional_args,
         native_array,
         fw,
     ):
-        input_dtype, x = dtype_and_x
+        input_dtype, input = dtype_and_input
+        assume("float16" not in input_dtype)
         helpers.test_frontend_function(
             input_dtypes=input_dtype,
             as_variable_flags=as_variable,
-            with_out=with_out,
+            with_out=False,
+            with_inplace=True,
```

Comment:
why are we hard-coding this to `True`? This should also be randomized by hypothesis.

Reply:
this has still not been resolved, it is still hard-coded to `True`?

Reply:
I have updated the tests I can update it

Reply:
yes that's what I meant, we should be randomly generating the value for `with_inplace` (which is the line of code this comment is pointing to), while the value for `with_out` should be hardcoded to `False`.